cleanup/cleanup.sh

```diff
@@ -1,3 +1,25 @@
-#!/bin/sh
+#!/bin/bash
 
-echo "Plugin called."
+set -e
+
+GIT_REPO_PATH=${KUBECTL_PLUGINS_LOCAL_FLAG_GIT_REPO_PATH}
+NAMESPACE=${KUBECTL_PLUGINS_GLOBAL_FLAG_NAMESPACE:-default}
+KEEP=${KUBECTL_PLUGINS_LOCAL_FLAG_KEEP}
+FROM_HEAD=$(( KEEP+1 ))
+
+# all commits except HEAD in the Git repository
+COMMIT_LIST=$(git -C ${GIT_REPO_PATH} rev-list --author-date-order HEAD | tail -n "+2")
+
+DELETION_CANDIDATES=$(oc get istag -n ${NAMESPACE} \
+    -o jsonpath='{.items[*].metadata.name}' \
+    --sort-by .metadata.creationTimestamp \
+    | tr ' ' '\n' | tail -n "+${FROM_HEAD}")
```

Comment:
Another way would be `jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'` to get the output you wanted